tutor_webhook_receiver/patches/k8s-services

```diff
@@ -0,0 +1,12 @@
+---
+apiVersion: v1
+kind: Service
+metadata:
+  name: webhook_receiver
+spec:
+  type: NodePort
```

Comment:
Doesn't defining `NodePort` here mean we can have only one instance of this service running on a single worker node? That will break once our Kubernetes cluster has more namespaces than worker nodes. I'd suggest to change this to `ClusterIP`.

Reply:
The little I understand about the effect of this suggests that you are right :slightly_smiling_face:  I'll change to `ClusterIP`.